modules/cert-manager-fips-support.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/index.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cert-manager-fips-support_{context}"]
+= About FIPS compliance for {cert-manager-operator}
+
+Red Hat {cert-manager-operator} is designed for FIPS. When running on {product-title} in FIPS mode, it uses the RHEL cryptographic libraries submitted to NIST for FIPS Validation on only the x86_64, ppc64le, and s390X architectures. For more information about the NIST validation program, see "cryptographic module validation program". For the latest NIST status for the individual versions of the RHEL cryptographic libraries submitted for validation, see "compliance activities and government standards". 
```

Comment:
Possible repetition of the word "Red Hat". Also, we need to include that FIPS support is only available in 1.14.0+.
```suggestion
{cert-manager-operator} is designed for FIPS starting v1.14.0 and above. When running on {product-title} in FIPS mode, it uses the RHEL cryptographic libraries submitted to NIST for FIPS Validation on only the x86_64, ppc64le, and s390X architectures. For more information about the NIST validation program, see "cryptographic module validation program". For the latest NIST status for the individual versions of the RHEL cryptographic libraries submitted for validation, see "compliance activities and government standards". 
```